components/camel-chatscript/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,33 @@
+## ---------------------------------------------------------------------------
```

Comment:
Please remove file, we are not using this logger

Reply:
done